### PR TITLE
fix test rop

### PR DESCRIPTION
**Pull request by zonca**
`test_rop.py`

### CODE REVIEW

**Comment by zonca** [comment]
@astrobc1 please review https://github.com/oirlab/liger_iris_pipeline/pull/68 first,
then I'll rebase so you don't have duplicated commits in the 2 pull request.

This PR fixes `test_rop.py`, this is a `Pipeline`, so this forced me to implement a workaround for pipelines due to issue #60

**Comment by zonca** [comment]
ok @astrobc1, now also the local Pipeline is un-necessary, this PR is ready for review and merge

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ jobs:
           C_INCLUDE_PATH: /usr/include/cfitsio/
     - name: Test
       run: |
-        source setup_local_crds.sh && pytest -v
+        source setup_local_crds.sh && pytest -v liger_iris_pipeline/tests/test_datamodels.py liger_iris_pipeline/tests/test_dark.py liger_iris_pipeline/tests/test_rop.py liger_iris_pipeline/tests/test_utils.py liger_iris_pipeline/tests/test_normalize.py
         #- name: Build docs
```
Comment (zonca) [inline .github/workflows/python-package.yml:45]: I am worried we could break some of the older tests if we do not test on CI.
So I suggest to add new tests here when we refactor them.
Once we are done we will set it back to `pytest -v` only.

Comment (zonca) [reply .github/workflows/python-package.yml:45]:
```
liger_iris_pipeline/tests/test_datamodels.py::test_load_iris_image_file PASSED [ 11%]
liger_iris_pipeline/tests/test_dark.py::test_dark_subtraction PASSED     [ 22%]
liger_iris_pipeline/tests/test_dark.py::test_dark_step PASSED            [ 33%]
liger_iris_pipeline/tests/test_rop.py::test_rop1 PASSED                  [ 44%]
liger_iris_pipeline/tests/test_rop.py::test_rop2 PASSED                  [ 55%]
liger_iris_pipeline/tests/test_normalize.py::test_normalize PASSED       [ 66%]
liger_iris_pipeline/tests/test_normalize.py::test_normalize_step PASSED  [ 77%]
liger_iris_pipeline/tests/test_assign_wcs.py::test_assign_wcs_step PASSED [ 88%]
liger_iris_pipeline/tests/test_asdf_schema.py::test_load_schema PASSED   [100%]
```

**Comment by zonca** [comment]
@astrobc1 oh yes, forgot, I had to update the crds cache

**Comment by zonca** [comment]
https://github.com/oirlab/liger-iris-crds-cache/commit/fea7631fc7cf81dd4f0113e17a767dd641665c3a

**Comment by astrobc1** [comment]
@zonca Is the [CI failing](https://github.com/oirlab/liger_iris_pipeline/actions/runs/10068442061/job/27931860646) a concern, or am I misinterpreting something here? The pytest command was the last line - I'm not sure what .sh file it is looking for.

**Comment by zonca** [comment]
I don't know. Let's ignore that for now

If keeps opening when I will look better into it

**Comment by zonca** [comment]
I don't know. Let's ignore that for now

It keeps happening then I will look better into it

**Pull request merged**